Binary files apt-2.1.20/.DS_Store and apt/.DS_Store differ
Binary files apt-2.1.20/apt-pkg/.DS_Store and apt/apt-pkg/.DS_Store differ
diff -urN apt-2.1.20/apt-pkg/contrib/configuration.cc apt/apt-pkg/contrib/configuration.cc
--- apt-2.1.20/apt-pkg/contrib/configuration.cc	2021-02-09 15:03:05.000000000 -0800
+++ apt/apt-pkg/contrib/configuration.cc	2021-02-13 02:26:57.000000000 -0800
@@ -39,7 +39,7 @@
 #include <string>
 #include <unordered_map>
 #include <vector>
-
+#include <array>
 #include <apti18n.h>
 
 using namespace std;
